data/gdshowsdb/1971.yaml

```diff
@@ -1330,7 +1330,85 @@
   :city: Austin
   :state: TX
   :country: US
-  :sets: []
+  :sets:
+  - :uuid: da6b040c-4916-4b4e-abb8-f3d56aa16169
+    :songs:
+    - :uuid: 8226ed8c-138e-4af5-b1e3-7c83c00f88cb
+      :name: Truckin'
+      :segued: false
+    - :uuid: c0246dee-5fae-4ad6-814d-a718d47371e8
+      :name: Bertha
+      :segued: false
+    - :uuid: ebb48d57-205e-4e6d-905b-930f1d3badcf
+      :name: Playing In The Band
+      :segued: false
+    - :uuid: b2201080-d9e7-444e-ba4e-95df64c14d57
+      :name: Deal
+      :segued: false
+    - :uuid: 5727375f-3d50-479b-8dce-1c391fec3ef3
+      :name: Jack Straw
+      :segued: false
+    - :uuid: d99f15ba-7d69-41c5-9ec0-5e1fe973ab4a
+      :name: Loser
+      :segued: false
+    - :uuid: 8144cd03-560a-4ef1-879e-070dadb5ced7
+      :name: Beat It On Down The Line
+      :segued: false
+    - :uuid: 0ead60da-6abf-4589-98dd-e56faca0c767
+      :name: Dark Star
+      :segued: true
+    - :uuid: c77efe2c-7f2f-4168-acb6-18d0f40913cb
+      :name: El Paso
+      :segued: true
+    - :uuid: 0ead60da-6abf-4589-98dd-e56faca0c767
```

Comment:
Deadlists has this as "Space" and Archive as "Jam". How about Jam for this one?

Reply:
Listened to it today.  Hints of Dark Star in the jam, calling it Dark Star Jam

    - :uuid: 0ead60da-6abf-4589-98dd-e56faca0c767
      :name: Dark Star
      :segued: true
    - :uuid: c77efe2c-7f2f-4168-acb6-18d0f40913cb
      :name: El Paso
      :segued: true
    - :uuid: 28138053-4eda-4f61-9e75-95f3a1341ffc
      :name: Dark Star Jam
      :segued: true